pytroll_collectors/s3stalker.py

```diff
@@ -54,6 +149,7 @@ class DatetimeHolder:
 
 def get_last_files(path, *args, pattern=None, **kwargs):
     """Get the last files from path (s3 bucket and directory)."""
+    kwargs['skip_instance_cache'] = True
```

Comment:
Um, what? Why is a kwarg modified here? Should the value be an instance attribute of the class? Then there could be a wrapper method that adjusts the attribute and calls this function.

Reply:
Any comments here?

Reply:
Do you remember why we needed this @TAlonglong ? 
The `skip_instance_cache` was apparently needed in order to get this to work when we polled for data in the bucket of @TAlonglong 
I think it is fair not to require it as user-configurable item, so don't want it in the config. I am unsure what you are after @pnuu ?

Reply:
I agree that this is strange. I can see that the kwargs are coming straight from the config file's `s3_kwargs`, so this should be a configuration item, not hardcoded here.